### ISSUE TITLE
[Bug][whitespace: check-operator] Doesn't allow empty tag in React Fragment.

### ISSUE THREAD

**Issue by zheeeng** [palantir/tslint#3632]
### Bug Report

- __TSLint version__: 5.8.0
- __TypeScript version__: 2.6.2
- __Running TSLint via__: VSCode

#### TypeScript code being linted

```ts
render() {
  return (
    <> // vscode tip: [tslint] missing whitespace (whitespace)
      <ChildA />
      <ChildB />
      <ChildC />
    </>
  );
}
```

with `tslint.json` configuration:

```json
{
    "rules": {
        "whitespace": [ true, "check-operator" ]
    }
}
```

**Comment by JoshuaKGoldberg** [comment]
@zheeeng I'm no longer able to reproduce this using the code you posted or similar versions. Looks like this was fixed! 🎉

```tsx
import * as React from "react";

class ChildA extends React.Component {
    render() {
        return <span />;
    }
}

const ChildB = ChildA;
const ChildC = ChildA;

class Foo extends React.Component {
    render() {
        return (
            <> // vscode tip: [tslint] missing whitespace (whitespace)
            <ChildA />
            <ChildB />
            <ChildC />
            </>
        );
    }
}
```
